Makefile

```diff
@@ -28,13 +32,13 @@ cluster-build:
 
 # TODO: Add deepcopy generation script/target
 .PHONY: generate
-generate: bindata crd
+generate: sync-ingress-crd bindata crd
 
 .PHONY: bindata
 bindata:
 	hack/update-generated-bindata.sh
 
-# Generate IngressController CRD from vendored API spec.
+# Generate DNSRecord CRD from internal API spec.
 .PHONY: crd
 crd:
 	hack/update-generated-crd.sh
```

Comment:
I'm not sure we need any new targets or scripts at all.

What if you add the following line to this `crd` target:

```
cp vendor/github.com/openshift/api/operator/v1/0000_50_ingress-operator_00-custom-resource-definition.yaml manifests/00-custom-resource-definition.yaml
```

And add the following line to the `verify-crd` target:

```
diff vendor/github.com/openshift/api/operator/v1/0000_50_ingress-operator_00-custom-resource-definition.yaml manifests/00-custom-resource-definition.yaml
```

Does that do the job?